rmgpy/tools/uncertainty.py

```diff
@@ -605,14 +605,22 @@ def sensitivityAnalysis(self, initialMoleFractions, sensitiveSpecies, T, P, term
         reactionSystem.attach(SimulationProfilePlotter(
             self.outputDirectory, reactionSystemIndex, self.speciesList))
         
+        simulatorSettings = SimulatorSettings() #defaults
+        
+        modelSettings = ModelSettings() #defaults
+        modelSettings.fluxToleranceMoveToCore = 0.1
```

Comment:
This was previously 1, does it have any affect since we're running sensitivity?

Reply:
If that matters I think we have much bigger problems than whether its set to 0.1 or 1.0.

Reply:
I guess I'm wondering if the reactor still does all the flux calculations and identifies species to be moved to the core, even though we aren't actually moving anything. A more direct question might be whether sensitivity analysis would be any faster with a tolerance of 1 vs 0.1?

Reply:
The edge is empty it doesn't have anything to check 0.1 or 1.0 against.

Reply:
Thanks, I forgot about that.